artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/QueueImpl.java

```diff
@@ -2550,6 +2552,11 @@ public boolean isExpirationRedundant() {
          return true;
       }
 
+      if (isInternalQueue() && name.toString().startsWith(MIRROR_ADDRESS)) {
+         logger.trace("Mirrored queues are not supposed to expiry. Address={}, Queue={}", address, name);
```

Comment:
Message is confusing, it rather reads like the origin queues shouldnt expire message, not just the mirror SnF queue.

Something like "Mirror SnF queues are not supposed to expire messages. Address={}, Queue={}" would be clearer.

Reply:
I was doing it for everything internal.. but a test failed.. and a previous JIRA asking exactly that behavior from the internal queue.. so I changed the implementation to deal with only Mirror. 

I just changed the title and commit name.. thanks